runtime/vm/MHInterpreter.inc

```diff
@@ -2558,3 +2559,4 @@ ffi_exit:
 	return ret;
 }
 #endif /* J9VM_OPT_PANAMA */
+#endif /* defined(J9VM_OPT_METHOD_HANDLE) */
```

Comment:
Does this file end with a newline?

Reply:
Yes.
![image](https://user-images.githubusercontent.com/83662958/118704529-39cc0d80-b7e5-11eb-9126-99caa5cfa47f.png)